catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -56,8 +57,7 @@
 
 	private static final StatusBarNotificationManager INSTANCE = new StatusBarNotificationManager();
 	public static final String CHANNEL_ID = "pocket_code_notification_channel_id";
-
-	private int notificationId;
+	private int notificationId = FOREGROUND_SERVICE;
```

Comment:
Why do we initialize notificationId with `FOREGROUND_SERVICE` here? What is the reason for doing that? In case this is done to avoid some unwanted behavior connected to the incrementing of notificationId when creating a new notifciation, we probably should consider a clearer approach or at least a clearer naming for this constant.